docs/resources/custom_domain.md

```diff
@@ -52,6 +52,9 @@ Import is supported using the following syntax:
 ```shell
 # Custom domains can be imported using their ID.
 #
+# You can find existing custom domain IDs using the auth0 management api
```

Comment:
Same comment as in https://github.com/auth0/terraform-provider-auth0/pull/462#discussion_r1095636495 applies here as well:) 

The docs folder is automatically generated. The import example script that needs to be changed is here: https://github.com/auth0/terraform-provider-auth0/blob/patch/contributing/examples/resources/auth0_custom_domain/import.sh and then run `make docs`.